[opencv4] Build failure

Package: opencv4[core,webp]:arm64-osx -> 4.8.0

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 576f3f1ad 2023-08-14 (12 hours ago)

**To Reproduce**

`vcpkg x-set-installed --enforce-port-checks --allow-unsupported opencv4[core,webp]`

**Failure logs**

```
-- Downloading https://github.com/opencv/opencv/commit/e5e1a3bfdea96bebda2ad963bc8f6cf17930aef7.patch -> opencv4-e5e1a3bfdea96bebda2ad963bc8f6cf17930aef7.patch...
-- Downloading https://github.com/opencv/opencv/archive/4.8.0.tar.gz -> opencv-opencv-4.8.0.tar.gz...
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/opencv-opencv-4.8.0.tar.gz
-- Applying patch 0001-disable-downloading.patch
-- Applying patch 0002-install-options.patch
-- Applying patch 0003-force-package-requirements.patch
-- Applying patch 0004-fix-eigen.patch
-- Applying patch 0005-fix-policy-CMP0057.patch
-- Applying patch 0006-fix-uwp.patch
-- Applying patch 0008-devendor-quirc.patch
-- Applying patch 0009-fix-protobuf.patch
-- Applying patch 0010-fix-uwp-tiff-imgcodecs.patch
-- Applying patch 0011-remove-python2.patch
-- Applying patch 0012-fix-zlib.patch
-- Applying patch 0015-fix-freetype.patch
-- Applying patch /Users/leanderSchulten/git_projekte/vcpkg/downloads/opencv4-e5e1a3bfdea96bebda2ad963bc8f6cf17930aef7.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean
-- Configuring arm64-osx
-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /opt/homebrew/Cellar/cmake/3.27.1/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/opencv4/portfile.cmake:455 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/install-arm64-osx-dbg-out.log</summary>

```
Change Dir: '/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg'

Run Build Command(s): /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/ninja/1.10.2-osx/ninja -v -v -j11 install
[1/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCAROTENE_NS=carotene_o4t -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/carotene/hal/../include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/carotene/hal -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -funwind-tables -fwrapv -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT 3rdparty/carotene/hal/CMakeFiles/tegra_hal.dir/dummy.cpp.o -MF 3rdparty/carotene/hal/CMakeFiles/tegra_hal.dir/dummy.cpp.o.d -o 3rdparty/carotene/hal/CMakeFiles/tegra_hal.dir/dummy.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/carotene/hal/dummy.cpp
[2/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCAROTENE_NS=carotene_o4t -DWITH_NEON -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/carotene/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -funwind-tables -fwrapv -Wno-unused-function -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT 3rdparty/carotene/hal/carotene/CMakeFiles/carotene_objs.dir/src/channels_combine.cpp.o -MF 3rdparty/carotene/hal/carotene/CMakeFiles/carotene_objs.dir/src/channels_combine.cpp.o.d -o 3rdparty/carotene/hal/carotene/CMakeFiles/carotene_objs.dir/src/channels_combine.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/3rdparty/carotene/src/channels_combine.cpp
...
Skipped 256 lines
...

[240/435] : && /opt/homebrew/Cellar/cmake/3.27.1/bin/cmake -E rm -f lib/libopencv_imgproc4d.a && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar qc lib/libopencv_imgproc4d.a  modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/accum.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/accum.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/approx.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/bilateral_filter.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/blend.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/box_filter.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/canny.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/clahe.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/color.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/color_hsv.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/color_lab.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/color_rgb.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/color_yuv.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/colormap.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/connectedcomponents.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/contours.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/convhull.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/corner.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/cornersubpix.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/demosaicing.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/deriv.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/distransform.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/drawing.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/emd.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/featureselect.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/filter.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/floodfill.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/gabor.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/generalized_hough.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/geometry.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/grabcut.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/hershey_fonts.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/histogram.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/hough.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/imgwarp.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/intelligent_scissors.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/intersection.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/linefit.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/lsd.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/main.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/matchcontours.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/median_blur.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/min_enclosing_triangle.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/moments.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/morph.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/phasecorr.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/pyramids.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/resize.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/rotcalipers.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/samplers.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/segmentation.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/shapedescr.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/smooth.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/spatialgradient.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/stackblur.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/subdivision2d.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/sumpixels.dispatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/tables.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/templmatch.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/thresh.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/src/utils.cpp.o modules/imgproc/CMakeFiles/opencv_imgproc.dir/opencl_kernels_imgproc.cpp.o && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib lib/libopencv_imgproc4d.a && /opt/homebrew/Cellar/cmake/3.27.1/bin/cmake -E touch lib/libopencv_imgproc4d.a && :
[241/435] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo && /opt/homebrew/Cellar/cmake/3.27.1/bin/cmake -DMODULE_NAME=photo -DCL_DIR=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/opencl -DOUTPUT=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo/opencl_kernels_photo.cpp -P /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/cmake/cl2cpp.cmake
[242/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_png.cpp.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_png.cpp.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_png.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/grfmt_png.cpp
[243/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_pam.cpp.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_pam.cpp.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_pam.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/grfmt_pam.cpp
[244/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_pfm.cpp.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_pfm.cpp.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_pfm.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/grfmt_pfm.cpp
[245/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fobjc-exceptions -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/apple_conversions.mm.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/apple_conversions.mm.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/apple_conversions.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/apple_conversions.mm
[246/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_spng.cpp.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_spng.cpp.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_spng.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/grfmt_spng.cpp
[247/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_pxm.cpp.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_pxm.cpp.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_pxm.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/grfmt_pxm.cpp
[248/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_sunras.cpp.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_sunras.cpp.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_sunras.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/grfmt_sunras.cpp
[249/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fobjc-exceptions -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/macosx_conversions.mm.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/macosx_conversions.mm.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/macosx_conversions.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/macosx_conversions.mm
[250/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_tiff.cpp.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_tiff.cpp.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_tiff.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/grfmt_tiff.cpp
[251/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_webp.cpp.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_webp.cpp.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_webp.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/grfmt_webp.cpp
FAILED: modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_webp.cpp.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DHAVE_IMGCODEC_HDR -DHAVE_IMGCODEC_PFM -DHAVE_IMGCODEC_PXM -DHAVE_IMGCODEC_SUNRASTER -DHAVE_WEBP -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/imgcodecs -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-deprecated-declarations -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_webp.cpp.o -MF modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_webp.cpp.o.d -o modules/imgcodecs/CMakeFiles/opencv_imgcodecs.dir/src/grfmt_webp.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/grfmt_webp.cpp
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgcodecs/src/grfmt_webp.cpp:47:10: fatal error: 'webp/decode.h' file not found
#include <webp/decode.h>
         ^~~~~~~~~~~~~~~
1 error generated.
[252/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/photo/CMakeFiles/opencv_photo.dir/src/align.cpp.o -MF modules/photo/CMakeFiles/opencv_photo.dir/src/align.cpp.o.d -o modules/photo/CMakeFiles/opencv_photo.dir/src/align.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/align.cpp
[253/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/photo/CMakeFiles/opencv_photo.dir/src/denoise_tvl1.cpp.o -MF modules/photo/CMakeFiles/opencv_photo.dir/src/denoise_tvl1.cpp.o.d -o modules/photo/CMakeFiles/opencv_photo.dir/src/denoise_tvl1.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/denoise_tvl1.cpp
[254/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/photo/CMakeFiles/opencv_photo.dir/src/calibrate.cpp.o -MF modules/photo/CMakeFiles/opencv_photo.dir/src/calibrate.cpp.o.d -o modules/photo/CMakeFiles/opencv_photo.dir/src/calibrate.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/calibrate.cpp
[255/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/photo/CMakeFiles/opencv_photo.dir/src/denoising.cuda.cpp.o -MF modules/photo/CMakeFiles/opencv_photo.dir/src/denoising.cuda.cpp.o.d -o modules/photo/CMakeFiles/opencv_photo.dir/src/denoising.cuda.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/denoising.cuda.cpp
[256/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/photo/CMakeFiles/opencv_photo.dir/src/contrast_preserve.cpp.o -MF modules/photo/CMakeFiles/opencv_photo.dir/src/contrast_preserve.cpp.o.d -o modules/photo/CMakeFiles/opencv_photo.dir/src/contrast_preserve.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/contrast_preserve.cpp
[257/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/photo/CMakeFiles/opencv_photo.dir/src/hdr_common.cpp.o -MF modules/photo/CMakeFiles/opencv_photo.dir/src/hdr_common.cpp.o.d -o modules/photo/CMakeFiles/opencv_photo.dir/src/hdr_common.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/hdr_common.cpp
[258/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/photo/CMakeFiles/opencv_photo.dir/src/inpaint.cpp.o -MF modules/photo/CMakeFiles/opencv_photo.dir/src/inpaint.cpp.o.d -o modules/photo/CMakeFiles/opencv_photo.dir/src/inpaint.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/inpaint.cpp
[259/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/photo/CMakeFiles/opencv_photo.dir/src/npr.cpp.o -MF modules/photo/CMakeFiles/opencv_photo.dir/src/npr.cpp.o.d -o modules/photo/CMakeFiles/opencv_photo.dir/src/npr.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/npr.cpp
[260/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/photo/CMakeFiles/opencv_photo.dir/src/merge.cpp.o -MF modules/photo/CMakeFiles/opencv_photo.dir/src/merge.cpp.o.d -o modules/photo/CMakeFiles/opencv_photo.dir/src/merge.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/merge.cpp
[261/435] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__OPENCV_BUILD=1 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/photo -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/imgproc/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG -std=c++11 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT modules/photo/CMakeFiles/opencv_photo.dir/src/denoising.cpp.o -MF modules/photo/CMakeFiles/opencv_photo.dir/src/denoising.cpp.o.d -o modules/photo/CMakeFiles/opencv_photo.dir/src/denoising.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.8.0-50001c828e.clean/modules/photo/src/denoising.cpp
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "gdal",
      "default-features": false,
      "features": [
        "poppler"
      ]
    }
  ],
  "$overrides": [
    {
      "name": "capstone",
      "version": "4.0.1-120373dc"
    }
  ],
  "$builtin-baseline": "338f99eed499b6df1628829e40695b396a70a1f7"
}

```
</details>

--- a/ports/libwebp/0011-fix-include.patch
+++ b/ports/libwebp/0011-fix-include.patch
@@ -1,0 +1,13 @@
+diff --git a/cmake/WebPConfig.cmake.in b/cmake/WebPConfig.cmake.in
+index 74a5891..4c084d9 100644
+--- a/cmake/WebPConfig.cmake.in
++++ b/cmake/WebPConfig.cmake.in
+@@ -11,7 +11,7 @@ endif()
+ include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
+ 
+ set_and_check(WebP_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
+-set(WEBP_INCLUDE_DIRS ${WebP_INCLUDE_DIRS})
++set(WEBP_INCLUDE_DIRS ${WebP_INCLUDE_DIR})
+ set(WebP_LIBRARIES "")
+ include(SelectLibraryConfigurations)
+ foreach(_vcpkg_libwebp_lib IN ITEMS @INSTALLED_LIBRARIES@ sharpyuv)

--- a/ports/libwebp/portfile.cmake
+++ b/ports/libwebp/portfile.cmake
@@ -9,6 +9,7 @@ vcpkg_from_github(
         0003-simd.patch
         0008-sdl.patch
         0010-fix_build.patch
+        0011-fix-include.patch
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS

--- a/ports/libwebp/vcpkg.json
+++ b/ports/libwebp/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libwebp",
   "version": "1.3.1",
-  "port-version": 1,
+  "port-version": 2,
   "description": "WebP codec: library to encode and decode images in WebP format",
   "homepage": "https://github.com/webmproject/libwebp",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4802,7 +4802,7 @@
     },
     "libwebp": {
       "baseline": "1.3.1",
-      "port-version": 1
+      "port-version": 2
     },
     "libwebsockets": {
       "baseline": "4.3.2",

--- a/versions/l-/libwebp.json
+++ b/versions/l-/libwebp.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "ce9a702064516d77e0b1239edc29efd603ee323e",
+      "version": "1.3.1",
+      "port-version": 2
+    },
+    {
       "git-tree": "42095edcaa6f9fb7a29822896d45b772f694a9cf",
       "version": "1.3.1",
       "port-version": 1

